.pre-commit-config.yaml

```diff
@@ -41,10 +41,10 @@ repos:
       - id: pydocstyle
         args:
           - --convention=numpy
-  # - repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
I found that mypy in the pre-commit was very slow. It's useful to have it temporarily enabled though.